app/src/main/res/layout/topic_overview_fragment.xml

```diff
@@ -2,17 +2,121 @@
 <layout
   xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto">
-  <androidx.constraintlayout.widget.ConstraintLayout
+  <data>
+    <variable
+      name="presenter"
+      type="org.oppia.app.topic.overview.TopicOverviewFragmentPresenter"/>
+    <variable
+      name="viewModel"
+      type="org.oppia.app.topic.overview.TopicOverviewViewModel"/>
+  </data>
+  <ScrollView
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
-    <TextView
-      android:id="@+id/dummy_text_view"
-      android:layout_width="wrap_content"
+    android:layout_height="match_parent"
+    android:overScrollMode="never"
+    android:scrollbars="none">
+    <androidx.constraintlayout.widget.ConstraintLayout
+      android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:text="This is dummy TextView for testing"
-      app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintEnd_toEndOf="parent"
-      app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toTopOf="parent"/>
-  </androidx.constraintlayout.widget.ConstraintLayout>
+      android:paddingBottom="24dp">
+      <ImageView
+        android:id="@+id/topic_thumbnail_image_view"
+        android:layout_width="0dp"
+        android:layout_height="200dp"
+        android:src="@{viewModel.topic.topicThumbnail.thumbnailGraphicValue}"
```

Comment:
@BenHenning 

I tried using `viewModel.topic.topicThumbnail.thumbnailGraphic` (type: LessonThumbnail) and `viewModel.topic.topicThumbnail.thumbnailGraphicValue` (type: Int).
In first case it failed because of `LessonThumbnail data type` and in second case it is not displaying image.

I took reference from: https://github.com/oppia/oppia-android/blob/0d8da66c7f68dfa718bddcfc3dc795bcac8a6a51/app/src/main/res/layout/topic_summary_view.xml#L23

Reply:
See #195 for the bindable adapter that you need to pull into this PR to make it work.

Reply:
This has been resolved with the help of #195